make/langtools/netbeans/nb-javac/build.xml

```diff
@@ -175,8 +182,17 @@
         <copy file="pom-nb-javac.xml" todir="build" />
 
         <replaceregexp match="(&lt;version&gt;).*(&lt;/version&gt;)"
-                       replace="\1${nb-javac-ver}-SNAPSHOT\2"
+                       replace="\1${maven.version}\2"
```

Comment:
Is there a reason to have to two configuration locations for the version?

Reply:
You are right, no, there is no reason for that. The `maven.version` shall be just internal variable derived from `nb-javac-ver` (with or without `-SNAPSHOT` suffix).

Reply:
README changed in fb4e079